src/openrct2/scenario/ScenarioSources.cpp

```diff
@@ -421,36 +421,41 @@ namespace OpenRCT2::ScenarioSources
         return false;
     }
 
-    void NormaliseName(utf8* buffer, size_t bufferSize, const utf8* name)
+    u8string NormaliseName(u8string_view input)
     {
-        size_t nameLength = String::LengthOf(name);
+        // American scenario titles should be converted to British name
+        // Don't worry, names will be translated using language packs later
+        for (const ScenarioAlias& alias : ScenarioAliases)
+        {
+            if (String::Equals(alias.Alternative, input))
+            {
+                LOG_VERBOSE("Found alias: %s; will treat as: %s", input, alias.Original);
+                return u8string(alias.Original);
+            }
+        }
+
+        size_t nameLength = input.length();
 
+        u8string normalisedName;
         // Strip "RCT(1|2)?" prefix off scenario names.
-        if (nameLength >= 3 && (name[0] == 'R' && name[1] == 'C' && name[2] == 'T'))
+        if (nameLength >= 3 && (input[0] == 'R' && input[1] == 'C' && input[2] == 'T'))
```

Comment:
We can just use [`.starts_with()`](https://en.cppreference.com/w/cpp/string/basic_string_view/starts_with) now, thanks to C++20.